src/lib/y2storage/resize_info.rb

```diff
@@ -32,8 +33,33 @@ module Y2Storage
   # This is a wrapper for Storage::ResizeInfo
   class ResizeInfo
     include StorageClassWrapper
+    include Yast::I18n
+    extend Yast::I18n
     wrap_class Storage::ResizeInfo
 
+    # rubocop:disable Metrics/LineLength
+    REASON_TEXTS =
+      {
+        RB_RESIZE_NOT_SUPPORTED_BY_DEVICE:       N_("Resizing is not supported by this device."),
+        RB_MIN_MAX_ERROR:                        N_("Combined limitations of partition and filesystem prevent resizing."),
+        RB_SHRINK_NOT_SUPPORTED_BY_FILESYSTEM:   N_("This filesystem does not support shrinking."),
+        RB_GROW_NOT_SUPPORTED_BY_FILESYSTEM:     N_("This filesystem does not support growing."),
+        RB_FILESYSTEM_INCONSISTENT:              N_("Filesystem consistency check failed."),
+        RB_MIN_SIZE_FOR_FILESYSTEM:              N_("This filesystem already has the minimum possible size."),
+        RB_MAX_SIZE_FOR_FILESYSTEM:              N_("This filesystem already has the maximum possible size."),
+        RB_FILESYSTEM_FULL:                      N_("The filesystem is full."),
+        RB_NO_SPACE_BEHIND_PARTITION:            N_("There is no space behind this partition."),
```

Comment:
In general I think we should help user to deal with it if we can so here write something like "There is no space behind this partition. If there is partition behind this one, then deleting should help."

Reply:
This is much too long at this point, and we would be lost in too many details very quickly. In this case, there might be another partition behind the one the user wanted to resize, or it might also be the end of the disk. This is not the place to do this kind of analysis; we just don't know any more than what the library gives us.

And besides, we are already very much overengineering this.

Reply:
Also consider that there might be a number of those messages that are reported at the same time, and we might run into screen space limitations (think about NCurses). Once we start giving more advanced tips, we'll need to make this scrollable which means RichText which means no good default for the message popup's size, so scrolling (in the worst case in both dimensions) would be the common thing.

And since the user needs to close the dialog to start fixing the problem, any more detailed instructions won't be visible on the screen for long so the user can refer to them later; this is the situation where less is really more (more useful).